Podfile

```diff
@@ -50,9 +50,9 @@ def wordpress_ui
 end
 
 def wordpress_kit
-  pod 'WordPressKit', '~> 11.0'
+  pod 'WordPressKit', '~> 12.0'
   # pod 'WordPressKit', git: 'https://github.com/wordpress-mobile/WordPressKit-iOS.git', commit: ''
-  # pod 'WordPressKit', git: 'https://github.com/wordpress-mobile/WordPressKit-iOS.git', branch: 'trunk'
+  # pod 'WordPressKit', git: 'https://github.com/wordpress-mobile/WordPressKit-iOS.git', branch: 'wordpress-com-rest-api-error-refactor'
```

Comment:
Please this line and line 144.

Reply:
Can you clarify what needs to be done regarding line 144? I'll merge this PR for now to unblock others. I'm happy to address any further issues.